lib/node_modules/@stdlib/math/base/special/normhermite-polynomial/lib/main.js

```diff
@@ -18,20 +18,70 @@
 
 'use strict';
 
+// MODULES //
+
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var isint = require( '@stdlib/math/base/assert/is-integer' );
+
+
 // MAIN //
 
 /**
 * Evaluates a normalized Hermite polynomial.
 *
-* @param {<TODO: type>} TODO - TODO: param desc
-* @returns {<TODO: type>} TODO: desc
+* @param {integer} n - non-negative polynomial degree
+* @param {number} x - evaluation point
+* @returns {number} function value
+*
+* @example
+* var v = normhermitepoly( 1, 0.5 );
+* // returns 0.5
+*
+* @example
+* var v = normhermitepoly( -1, 0.5 );
+* // returns NaN
 *
 * @example
-* var v = normhermitepoly( 1, 1.0 );
-* // returns TODO
+* var v = normhermitepoly( 0, 0.5 );
+* // returns 1.0
+*
+* @example
+* var v = normhermitepoly( 2, 0.5 );
+* // returns -0.75
 */
-function normhermitepoly() {
-	// TODO: implementation
+function normhermitepoly( n, x ) {
+	var y1;
+	var y2;
+	var y3;
+	var i;
+
+	if ( isnan( n ) || isnan( x ) ) {
+		return NaN;
+	}
+	if ( !isint( n ) ) {
+		return NaN;
+	}
+	if ( n < 0 ) {
+		return NaN;
+	}
+	if ( n === 0 ) {
+		// The x gets completely cancelled from the expression
+		return 1.0;
+	}
+	if ( n < 0 ) {
```

Comment:
This should be removed, as the condition is addressed at L64. To catch such dead code, you can generate a local test coverage report as follows:

```bash
$ make test-cov TESTS_FILTER=.*/normhermite-polynomial/.*
$ make view-cov
```

Reply:
Thanks for the tip on the dead code elimination.